### ISSUE TITLE
is it possible that forbiding glide get command to update those already installed packages?

### ISSUE THREAD

**Issue by redhatlinux10** [Masterminds/glide#450]
glide : 0.10.2
os : windows 7 32bit

this is my circumstance:
in my codebase, i have installed `gin-gonic` through command `glide get`. when i try to install  `go-i18n`
through command `glide get github.com/nicksnyder/go-i18n`, the command try to update gin-gonic as well. is it possible to disable it ? maybe a switch should be provided.
>glide get github.com/gin-gonic/gin
[INFO] Preparing to install 1 package.
[INFO] Importing github.com/gin-gonic/gin
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/gin-gonic/gin.
[INFO] Resolving imports
[INFO] Downloading dependencies. Please wait...
glide get github.com/nicksnyder/go-i18n
[INFO] Preparing to install 1 package.
[INFO] Importing github.com/nicksnyder/go-i18n
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/nicksnyder/go-i18n.
[INFO] Fetching updates for github.com/gin-gonic/gin.
[INFO] Resolving imports
[INFO] Downloading dependencies. Please wait...

**Comment by sdboyer** [comment]
yep, this is a known issue, and it's a goal to support not updating everything at once. see #252 (among others).

**Comment by lming** [comment]
+1. Getting a new package should not automatically update all existing packages ...
